### ISSUE TITLE
_static/terminado.js and uimod_embed.js missing from distribution on PyPI

### ISSUE THREAD

**Issue by asmodehn** [jupyter/terminado#83]
Check: https://pypi.org/project/terminado/#files

This lacks :
- https://github.com/jupyter/terminado/blob/master/terminado/_static/terminado.js
- https://github.com/jupyter/terminado/blob/master/terminado/uimod_embed.js

Is it intended ?
Because of this, currently, running the demo code outside the repo breaks...
```
ERROR:tornado.general:Could not open static file '/home/alexv/.local/share/virtualenvs/terminado-A6GpWQ5Q/lib/python3.8/site-packages/terminado/_static/terminado.js'
```

**Comment by wsowens** [comment]
I confirm that I had this same issue. As a temporary workaround, I downloaded the files and made symlinks, but hopefully this can be fixed.

**Comment by MircoT** [comment]
Same for me, the package installed with `setup.py` has no `_static` folder and the other `js` file.

**Comment by blink1073** [comment]
Fixed by #87
